__init__.py

```diff
@@ -23,12 +30,21 @@ def __init__(self, config):
         except KeyError as e:
             _LOGGER.error("Missing auth token! You must set 'github-token' in your config")
         self.name = self.config.get("name", "github")
+        # had troubles using the python 3.6 syntax, using asyncio as the package
+        loop = asyncio.get_event_loop()
```

Comment:
You can use `self.opsdroid. eventloop` instead of getting it yourself.

Reply:
Actually I see you're running this in the `__init__` so that wont be populated yet. Can you move the whole section into the `connect` method